peachjam_api/views.py

```diff
@@ -127,3 +131,44 @@ def post(self, request):
                 )
                 return Response(html)
         return Response()
+
+
+class DocumentProblemView(APIView):
```

Comment:
Make this a regular `FormView` view, and have a `DocumentProblemForm` class that has the various fields that get validated.